pkg/datapath/linux/ethtool/ethtool.go

```diff
@@ -90,10 +86,5 @@ func IsVirtualDriver(iface string) (bool, error) {
 		return false, err
 	}
 
-	switch drvName {
-	case veth:
-		return true, nil
-	}
-
-	return true, nil
+	return drvName == "veth", nil
```

Comment:
Thanks I missed converting this last line when changing from `NotIsVritualDriver -> IsVirtualDriver` and clearly broke this. Thanks a lot for the catch.

Reply:
The switch was originally to support multiple virtual devices, but for now I think veth is good enough because this is guarded to only be used on EKS.